### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by danigalindo10**
### Created Add and Remove function ✅ ✅ ✅ 
For this project, I implemented the following changes:
- I removed all hardcoded items from the tasks array.
- I created a new JavaScript file for the new functionality.
- I implemented a function for adding a new task.
- I implemented a function for deleting a task.
- I implemented a function for editing task descriptions.

### CODE REVIEW

File: src/modules/edit-task.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+// Functions
+const editTask = (index, description = '', completed = false) => {
+  if (description) {
+    const toDoTasks = getData();
+    toDoTasks[index - 1].description = description;
+    toDoTasks[index - 1].completed = completed;
+    localStorage.setItem('toDoData', JSON.stringify(toDoTasks));
+  }
+};
```
Comment (omarsalem7) [inline src/modules/edit-task.js:11]: - [x] Kindly check your local storage because after editing and refreshing the page the updated todo vanished.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @danigalindo10 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Add and remove todo work correctly✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/edit-task.js (hunk 1 above)
Comment (danigalindo10) [reply src/modules/edit-task.js:11]: Thank you @omarsalem7,
I've just applied all the required changes. Thank you for your feedback. 🙂

**Review by codecaiine — APPROVED** [review]
Hi @danigalindo10 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**